LeaderboardBackend/appsettings.Staging.json

```diff
@@ -0,0 +1,26 @@
+{
+	"Logging": {
+		"LogLevel": {
+			"Default": "Information",
+			"Microsoft.AspNetCore": "Warning"
+		}
+	},
+	"EnvPath": ".env",
+	"Jwt": {
+		"Key": "coolsecretkeywow",
+		"Issuer": "leaderboards.gg"
+	},
```

Comment:
We'll really need to change this ha

Reply:
In dev and staging it doesn't really matter, but we should probably put at least a cryptographically generated key for consistency with eventual production 😄